cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java

```diff
@@ -82,43 +83,85 @@ public Integer call() throws Exception {
-                if (input.equals("y") || input.equals("yes")) {
-                    doRemoveServiceInstance(yamlCommandOptions);
-                } else {
-                    System.out.println("Delete cancelled");
-                    
+
+                // turn into lower case, we would accept an upper case response, too
+                input = input.toLowerCase();
+
+                if (!(input.equals("y") || input.equals("yes"))) {
+                    System.out.println("Cancelled by user");
                     return 1;
                 }
             }
+
+            // try to remove all services on a best-effort basis
+            // the method should handle errors and log them appropriately
+            doRemoveServiceInstance(services);
             
             return 0;
         }
 
-        private void doRemoveServiceInstance(YamlCommandOptions yamlCommandOptions) throws Exception {
-            Log.info("Removing services...");
+        private Map<String, ServiceBean> readServicesFromYamlFile() throws IOException {
             SpecBean specBean = YamlMapper.loadBean(yamlCommandOptions.getYamlFilePath(), SpecBean.class);
-            Map<String, ServiceBean> serviceBeans = specBean.getServices();
+            return specBean.getServices();
+        }
+
+        /**
+         * Try to remove service instances. This method handles potential errors internally by logging them.
+         * In case it fails, it returns false. The caller can opt to use this to perform additional error handling.
+         * @param services services to remove
+         * @return true on success, false if there were errors
+         */
+        private boolean doRemoveServiceInstance(Map<String, ServiceBean> services) {
+            Log.info("Removing services...");
+
+            DefaultCloudFoundryOperations cfOperations;
+
+            try {
+                cfOperations = CfOperationsCreator.createCfOperations(loginOptions);
+            } catch (Exception e) {
+                Log.error("Failed to create CF operations: ", e.getMessage());
+                return false;
+            }
 
-            DefaultCloudFoundryOperations cfOperations = CfOperationsCreator.createCfOperations(loginOptions);
             ServicesOperations servicesOperations = new ServicesOperations(cfOperations);
 
-            for (Entry<String, ServiceBean> serviceEntry : serviceBeans.entrySet()) {
+            // I'm optimistic at the beginning of this loop
+            // prove me wrong!
+            boolean success = true;
+
+            for (Entry<String, ServiceBean> serviceEntry : services.entrySet()) {
                 String serviceName = serviceEntry.getKey();
-                servicesOperations.removeServiceInstance(serviceName);
+
+                try {
+                    servicesOperations.removeServiceInstance(serviceName);
+                } catch (Exception e) {
+                    Log.error("Failed to remove service ", serviceName, ": ", e.getMessage());
```

Comment:
In my opinion logging a warning would be more adequate because the program is not aborted anymore.

Reply:
I think a warning is more appropriate when the issue can be fixed by the program, e.g., when providing a `--force` flag. But that's not the case here. Apparently, to resolve the issue, you need to open up the CF CLI or use the webinterface, right? I mean, the method also returns `false` for that purpose; I like to give feedback about errors.

Reply:
Okay you convinced me